airflow/www_rbac/templates/airflow/version.html

```diff
@@ -20,14 +20,37 @@
 #}
 
 {% extends base_template %}
-
 {% block content %}
     {{ super() }}
-    <h2>{{ title }}</h2>
-  {% set version_label = 'Version' %}
-    {% if airflow_version %}
-        <h4>{{ version_label }} : {{ airflow_version }}</a></h4>
-    {% else %}
-        <h4>{{ version_label }} : Not Available</h4>
-    {% endif %}
+    <h1>Changelog</h1> 
+	<br>
+	<div style="overflow-y:scroll; height:400px; border-style: groove; padding-left:15px; padding-right:15px">
+	{% for version, headings in data.items() %}
```

Comment:
Change variable names. Suggestion: Change `headings` to `changelog_data`,  `data` to `changelogs`.